packages/@lwc/integration-karma/scripts/karma-configs/hydration/sauce.js

```diff
@@ -18,18 +18,26 @@ const SAUCE_BROWSERS = [
     {
         label: 'sl_chrome_latest',
         browserName: 'chrome',
-        version: 'latest',
+        browserVersion: 'latest',
+    },
+    // TODO [#3083]: Update to latest firefox and geckodriver.
+    // Pin firefox version to 105 and geckodriver to 0.30.0 for now because of
+    // issues running the latest version of firefox with geckodriver > 0.30.0
+    // in saucelabs.
+    // https://saucelabs.com/blog/update-firefox-tests-before-oct-4-2022
+    {
+        label: 'sl_firefox_latest',
+        browserName: 'firefox',
+        browserVersion: '105',
+        sauceOptions: {
+            geckodriverVersion: '0.30.0',
+        },
     },
-    // TODO [#3083]: re-enable Firefox tests
-    // {
-    //     label: 'sl_firefox_latest',
-    //     browserName: 'firefox',
-    //     version: '103',
-    // },
     {
         label: 'sl_safari_latest',
         browserName: 'safari',
-        version: 'latest',
+        browserVersion: 'latest',
+        platformName: 'macOS 12',
     },
```

Comment:
These changes were made to change the format to W3C, JWP is now deprecated.

See https://saucelabs.com/platform/platform-configurator for more details.